index.bs

```diff
@@ -2820,17 +2821,78 @@ For [=prompt=]:
 
 For [=window open steps=]:
 
-14. If targetNavigable is null, then return null.
+1. <ins>Let |windowOpenDisposition| be "{{WindowOpenDisposition/ignore}}".</ins>
+
+2. <ins>Let |embedderParent| be <var ignore=''>sourceDocument</var>'s [=node
+    navigable=]'s [=embedderParent=].</ins>
+
+<del>1. If the event loop's termination nesting level is nonzero, then return
+    null.</del>
+
+3. <ins>If the event loop's termination nesting level is nonzero:</ins>
+
+      1. <ins>If |embedderParent| is not null:</ins>
+
+            1. <ins>[=Fire a "newwindow" event=] with |embedderParent|, |url|,
+                |target|, null, and |windowOpenDisposition|.</ins>
+
+      1. <ins>Return null.<ins>
+
+...
+
+<del>14. If |targetNavigable| is null, then return null.</del>
+
+14. <ins>If |targetNavigable| is null:</ins>
+
+      1. <ins>If |embedderParent| is not null:</ins>
+
+            1. <ins>[=Fire a "newwindow" event=] with |embedderParent|, |url|,
+                |target|, |targetNavigable|, and |windowOpenDisposition|.</ins>
+
+      1. <ins>Return null.<ins>
+
+15. If windowType is either "new and unrestricted" or "new with no opener",
+    then:
+
+      1. <ins>Set |windowOpenDisposition| to one of
+          "{{WindowOpenDisposition/new_background_tab}}",
+          "{{WindowOpenDisposition/new_foreground_tab}}",
+          or "{{WindowOpenDisposition/new_window}}".</ins>
+
+      Note: The value depends on the behavior support of the user agent and user
+          settings.
+
+      1. Set |targetNavigable|'s active browsing context's is popup to the
+          result of checking if a popup window is requested, given
+          tokenizedFeatures.
+
+      1. <ins>If |targetNavigable|'s active browsing context's is popup is true,
```

Comment:
https://respec.org/xref/?term=active+browsing+context, [=is popup=], `true`
https://html.spec.whatwg.org/multipage/document-sequences.html#is-popup